packages/plugin-html-keyboard-response/src/index.ts

```diff
@@ -152,6 +152,60 @@ class HtmlKeyboardResponsePlugin implements JsPsychPlugin<Info> {
       this.jsPsych.pluginAPI.setTimeout(end_trial, trial.trial_duration);
     }
   }
+
+  simulate(
+    trial: TrialType<Info>,
+    simulation_mode,
+    simulation_options: any,
+    load_callback: () => void
+  ) {
+    if (simulation_mode == "data-only") {
+      load_callback();
+      this.simulation_data_mode(trial, simulation_options);
```

Comment:
I'd prefer `simulate_data` and `simulate_visual` to keep the function names imperative...